src/graphics.rs

```diff
@@ -1628,6 +1634,263 @@ impl Drawable for Sprite {
     }
 }
 
+// AsciiSubset refers to the subset of ascii characters which give alphanumeric characters plus symbols
+pub enum CharacterListType {
+    AsciiSubset,
+    Ascii,
+    Cyrillic,
+    Thai,
+    Vietnamese,
+    Chinese,
+    Japanese,
+}
```

Comment:
"set" enum should probably implement `Debug`, `Clone`, `Copy`, and perhaps `PartialEq` and `Eq` for ease of comparison if ever necessary. i'm tempted to kitchen-sink it and also derive `PartialOrd`, `Ord`, and `Hash` as well